components/script/script_thread.rs

```diff
@@ -2872,6 +3147,10 @@ impl ScriptThread {
         RemoteEventTaskSource(self.remote_event_task_sender.clone(), pipeline_id)
     }
 
+    pub fn rendering_task_source(&self, pipeline_id: PipelineId) -> RenderingTaskSource {
```

Comment:
I think this doesn't need to be public and definitely not public outside the crate.